<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_user</class>
 <widget class="QDialog" name="Dialog_user">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Benutzer verwalten</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QListWidget" name="listWidget_names">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>265</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="7" column="0" colspan="2">
     <widget class="QCheckBox" name="checkBox_modul_projekte">
      <property name="text">
       <string>Zugang zu Modul Projekte</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QCheckBox" name="checkBox_modul_lager">
      <property name="text">
       <string>Zugang zu Modul Lager</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_name"/>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QCheckBox" name="checkBox_isadmin">
      <property name="text">
       <string>hat Administrator-Rechte</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Passwort</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="spinBox_id">
      <property name="maximum">
       <number>999999</number>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QCheckBox" name="checkBox_modul_lieferanten">
      <property name="text">
       <string>Zugang zu Modul Lieferanten</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_name">
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QCheckBox" name="checkBox_modul_artikel">
      <property name="text">
       <string>Zugang zu Modul Artikel</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="2">
     <widget class="QCheckBox" name="checkBox_function_artfavsort">
      <property name="text">
       <string>Darf Artikel die Favoriten sind sortieren</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_pwd"/>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QCheckBox" name="checkBox_modul_personal">
      <property name="text">
       <string>Zugang zu Modul Personal</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="2">
     <widget class="QCheckBox" name="checkBox_modul_matlist">
      <property name="text">
       <string>Zugang zu Modul Materialliste</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0" colspan="2">
     <widget class="QCheckBox" name="checkBox_function_projektdel">
      <property name="text">
       <string>Darf Projekte löschen</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>147</x>
     <y>410</y>
     <width>271</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_ok">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_cancel">
      <property name="text">
       <string>Abbrechen</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>261</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButton_newuser">
      <property name="text">
       <string>neuer Benutzer</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_deleteuser">
      <property name="text">
       <string>Benutzer löschen</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>listWidget_names</tabstop>
  <tabstop>lineEdit_name</tabstop>
  <tabstop>lineEdit_pwd</tabstop>
  <tabstop>spinBox_id</tabstop>
  <tabstop>checkBox_isadmin</tabstop>
  <tabstop>checkBox_modul_artikel</tabstop>
  <tabstop>checkBox_modul_lieferanten</tabstop>
  <tabstop>checkBox_modul_lager</tabstop>
  <tabstop>checkBox_modul_projekte</tabstop>
  <tabstop>checkBox_modul_personal</tabstop>
  <tabstop>checkBox_modul_matlist</tabstop>
  <tabstop>pushButton_ok</tabstop>
  <tabstop>pushButton_cancel</tabstop>
  <tabstop>pushButton_deleteuser</tabstop>
  <tabstop>pushButton_newuser</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
